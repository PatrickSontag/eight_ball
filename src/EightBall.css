.app-container {
    display: flex;
    align-items: center;
    justify-content: center;
    background-color: rgb(240, 191, 191);
    height: 100vh;
    position: relative;
}

.eightball {
    width: 400px;
    height: 400px;
    border-radius: 50%;
    background-color: black;
    display: flex;
    align-items: center;
    justify-content: center;
}

.ball-text {
    color: white;
    font-size: 25px;
    position: absolute;
}