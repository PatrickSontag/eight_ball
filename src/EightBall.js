import React, { useState } from "react";
import ANSWERS from "./answers";

const EightBall = () => {
    return(
        <div>
            <div class="circle">
                <div class="ball-text">Think of a Question</div>
            </div>
        </div>
    )
}

export default EightBall;