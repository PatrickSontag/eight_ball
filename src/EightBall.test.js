import React from 'react';
import { render, screen } from '@testing-library/react';
import EightBall from './EightBall';

test('it rendersnwiyhout crashing', () => {
  render(<EightBall />);
});

test('it matches snapshot', () => {
    const {asFragment} = render(<EightBall />);
    expect(asFragment()).toMatchSnapshot()
})