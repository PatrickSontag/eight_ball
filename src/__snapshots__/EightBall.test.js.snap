// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="title"
  >
    MAGIC EIGHTBALL
  </div>
  <div
    class="app-container"
  >
    <button
      class="btn"
    >
      <div
        class="eightball black"
      >
        <div
          class="ball-text"
        >
          Think of a Question
        </div>
      </div>
    </button>
  </div>
</DocumentFragment>
`;
